lib/driver.js

```diff
@@ -435,6 +442,43 @@ class AndroidDriver extends BaseDriver {
     }
   }
 
+  async setSharedPreferences () {
+    let sharedPrefs = this.opts.sharedPreferences;
```

Comment:
@jlipps no, it takes the sharedPrefs object and creates an _.xml_ file
https://github.com/vrunoa/shared-preferences-builder/blob/master/lib/index.js#L69-L82.